test/integration_test_helper.rb

```diff
@@ -20,18 +20,49 @@
 end
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
-Capybara.register_driver :selenium_chrome do |app|
+
+javascript_driver = ENV.fetch("JS_TEST_DRIVER") { ENV['DEBUG_JS_TEST'] ? :selenium_chrome : :selenium_chrome_headless }.to_sym
+
+def get_chrome_options
   options = Selenium::WebDriver::Chrome::Options.new
-  options.args << '--disable-gpu'
-  options.args << '--no-sandbox'
-  options.args << '--window-size=1024,768'
-  options.args << '--headless' unless ENV['DEBUG_JS_TEST'] == '1'
-  Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
+  additional_options = ENV.fetch('ADDITIONAL_CHROME_OPTIONS', '')
+  additional_options.split(";").each do |add_opt|
+    options.args << add_opt
+  end
+  options
+end
+
+if javascript_driver == :selenium_chrome_remote
+  ShowMeTheCookies.register_adapter(:selenium_chrome_remote, ShowMeTheCookies::SeleniumChrome)
+
+  Capybara.register_driver :selenium_chrome_remote do |app|
+    selenium_remote_host = ENV.fetch('SELENIUM_REMOTE_HOST', nil)
+    selenium_remote_port = ENV.fetch('SELENIUM_REMOTE_PORT', 4444)
+    Capybara::Selenium::Driver.new(
+      app,
+      browser: :remote,
+      url: "http://#{selenium_remote_host}:#{selenium_remote_port}/wd/hub",
+      options: get_chrome_options)
+  end
 end
+
 Capybara.configure do |config|
-  config.javascript_driver      = ENV["JS_TEST_DRIVER"]&.to_sym || :selenium_chrome
-  config.default_max_wait_time  = 20
+  config.javascript_driver = javascript_driver
+  config.default_max_wait_time = 20
   config.enable_aria_label = true
+  if ENV.fetch("JS_TEST_DRIVER", nil) == :selenium_chrome_remote.to_s
```

Comment:
```suggestion
  if ENV.fetch("JS_TEST_DRIVER", nil) == 'selenium_chrome_remote'
```

Reply:
IMHO it is clearer to derive the string from the symbol instead of kind of duplicating the definition.

Reply:
I disagree. While it's minimal, you'd adding redundancy. If it was a constant, I'd agree. Now there's just additional mental processing required when you read this that it does in fact ends up being a string.

Reply:
Okay then. I changed it.